fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java

```diff
@@ -1158,6 +1181,41 @@ public TStreamLoadPutResult streamLoadPut(TStreamLoadPutRequest request) {
         return result;
     }
 
+    private void streamLoadPutWithSqlImpl(TStreamLoadPutRequest request) throws UserException {
+        String loadSql = request.getLoadSql();
```

Comment:
Add a log here to indicate that we receive a load request.

Reply:
> Add a log here to indicate that we receive a load request.

OK